Automattic/jetpack
# Social Logos: update to the most recent version of the package

## Proposed changes:

This includes 2 things:

- Fixes to the X and Threads logos, to avoid invalid SVG warnings (see https://github.com/Automattic/social-logos/pull/127#discussion_r1356363587 ).
- An updated X logo to match updated branding guidelines (in practice, the logo is now aligned a bit differently on the grid).

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

**Testing sharing buttons**

* Go to Jetpack > Settings > Social and enable sharing buttons.
* Go to Settings > Sharing and add some sharing buttons to your site. Use the icon-only sharing type.
* The buttons should look good in the admin preview, as well as on the frontend when your changes are saved. **Make sure all buttons look good, not just the X one.**

**Testing the post-publish modal**

* Create a new post and hit publish or for an existing post, run this in browser console
    - `wp.data.dispatch('core/edit-post').togglePublishSidebar();`
* On the post publish panel, confirm that you see the X icon; it should look good
* Confirm that it works as expected
* Open Jetpack sidebar
* Under "Share this post", click on the Quick Share icon
* Confirm that you see the X icon
* Confirm that you see no warnings in your JS console.
